modules/auxiliary/scanner/ftp/ftp_login.rb

```diff
@@ -70,6 +70,7 @@ def run_host(ip)
         max_send_size: datastore['TCP::max_send_size'],
         send_delay: datastore['TCP::send_delay'],
         connection_timeout: 30,
```

Comment:
Maybe make this user-mutable

Reply:
`ConnectTimeout` might work here as an existing datastore option to use

Reply:
The default value for `ConnectTimeout` is `10`. Do we want to make a separate configurable timeout datastore value for this module? If so, I can add that in before landing.

Reply:
I think that makes sense.